<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>710</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>710</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>710</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Главная</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/logo.png</normaloff>../img/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: rgb(247, 252, 255);
	font: 12pt &quot;Arial&quot;;
}
QPushButton {
	background-color: rgb(51, 177, 255);
	color: rgb(255, 255, 255);
	border: 3px solid #ffffff;
	border-radius: 17px;
	font: 14pt &quot;Arial&quot;;
}
QPushButton:hover {
	border-radius: 19px;
	background-color: rgb(52, 202, 232);
}
QPushButton:pressed {
	border-radius: 20px;
	background-color: rgb(0, 222, 0);
}
QLineEdit {
	border-radius: 12px;
	border: 1px solid rgb(51, 177, 255);
}
QLineEdit:hover {
	border-radius: 15px;
	background-color: rgb(239, 239, 239);
	border: 1px solid rgb(0, 222, 0);
}
QComboBox {
	border-radius: 12px;	
}
QSpinBox {
	border-radius: 12px;	
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>300</width>
      <height>70</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Выгрузить файлы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>621</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>Управление файлами</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>260</y>
      <width>300</width>
      <height>70</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>Добавить файлы</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
